website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -37,6 +37,11 @@ cd $HOME/projects/memcached-operator
 # so all API groups will be <group>.example.com
 operator-sdk init --domain example.com --repo github.com/example/memcached-operator
 ```
+`--domain` will be used as the prefix of the API group your custom resources will be created in.
+API groups are a mechanism to group portions of the Kubernetes API. You're probably already familiar with
+some of the core Kubernetes API groups, such as `apps` or `rbac.authorization.k8s.io`. You should name your
+domain to group your resource types in meaningful group(s), both for ease of understanding and because these
+groups determine how access can be controlled to your resource types using RBAC. For more information, see [the core Kubernetes docs](https://kubernetes.io/docs/reference/using-api/#api-groups) and [the Kubebuilder docs](https://book.kubebuilder.io/cronjob-tutorial/gvks.html).
```

Comment:
@jberkhahn IMO mentioning RBAC isn't necessary since that's not the only thing for which we have `groups`. 
Rest looks good, we can merge it. 

```suggestion
domain to group your resource types in meaningful group(s). For more information, see [the core Kubernetes docs](https://kubernetes.io/docs/reference/using-api/#api-groups) and [the Kubebuilder docs](https://book.kubebuilder.io/cronjob-tutorial/gvks.html).
```